spec/view_helper/description_spec.rb

```diff
@@ -47,6 +47,12 @@
     end
   end
 
+  it 'should change double quote to single quote' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.